master/custom/builders.py

```diff
@@ -300,6 +300,8 @@
     # FreBSD x86-64 clang
     # FreeBSD 15 is CURRENT: development branch (at 2023-10-17)
     ("AMD64 FreeBSD15", "opsec-fbsd15", UnixBuild),
+
+    ("ARM Raspbian", "stan-raspbian", UnixBuild),
```

Comment:
You should find another name, otherwise there are errors:

```

Configuration Errors:
  duplicate builder name 'ARM Raspbian 3.x'
  duplicate builder name 'ARM Raspbian 3.13'
  duplicate builder name 'ARM Raspbian 3.12'
  duplicate builder name 'ARM Raspbian 3.11'
  duplicate builder name 'ARM Raspbian 3.10'
  duplicate builder name 'ARM Raspbian 3.9'
  duplicate builder name 'ARM Raspbian PR'
```